marshalable-derive/src/lib.rs

```diff
@@ -66,21 +66,22 @@ fn derive_tpm_marshal_inner(input: DeriveInput) -> Result<TokenStream> {
             ));
         }
     };
-
     let expanded = quote! {
-        #pure_impl
-        // The generated impl.
-        impl Marshalable for #name  {
-            fn try_unmarshal(buffer: &mut UnmarshalBuf) -> tpm2_rs_marshalable::exports::errors::TpmRcResult<Self> {
-                #unmarsh_text
-            }
+        const _: () = {
```

Comment:
the anonymous module is not necessary, or am I missing something?

Reply:
I suppose it's no longer necessary, I can remove it - it would protect us from future refactors adding helper functions and accidentally exposing them again, but we currently aren't creating any more of such functions.

I can remove it.